.github/workflows/release_droid_upload_github_release_assets.yml

```diff
@@ -29,9 +29,6 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Run tests
-        uses: ./.github/actions/run_tests
-
```

Comment:
unit tests we still want to run in github for quicker feedback without logging in into AWS